xbmc/platform/win32/WinMain.cpp

```diff
@@ -88,6 +89,31 @@ static std::shared_ptr<CAppParams> ParseCommandLine()
   return appParamParser.GetAppParams();
 }
 
+static bool CheckAndSetFileFlag(bool usePlatformDirectories, HANDLE& handle)
```

Comment:
May I suggest to put this function into an unnamed namespace instead of tagging it static?

Reply:
You may :-) I followed the style of the existing code, there already are a few static functions in that .cpp.
I could add a commit to modernize I guess.

Reply:
> I followed the style of the existing code

Our coding guidelines are clear to that regard: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#62-local-functions + "The coding guidelines should be met by every code change, be it editing existing code, adding new code to existing source files, or adding completely new source files." :-)